mysql-test/suite/clone/t/local_create_synchronization_coordinates.test

```diff
@@ -0,0 +1,103 @@
+# Test that after a local clone command, synchronization_coordinates file is created
+# containing 4 key/val pairs, we only examine 3 here
+# excluding gtid get from binlog_file/offset
+
+--source ../include/clone_connection_begin.inc
+
+# Populate gtid
+SET @@GLOBAL.ENFORCE_GTID_CONSISTENCY=ON;
```

Comment:
Ideally the testcases have to support arbitrary `--mysqld=--any-server-option=value` run arguments. This testcase requires GTID, so in general case we do the pattern of saving the start global value and restoring at the end. Since `gtid_mode` is a special in that setting it goes through `_permissive` stages, in this case it is simpler to gate the testcase with `--source have_gtid.inc`, and adding a testcase option file with `--gtid-mode=ON`